src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery.csproj

```diff
@@ -169,10 +170,6 @@
     <None Include="StartupScripts\*.ps1" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\Profile\Commands.Profile\Commands.Profile.csproj">
```

Comment:
Why remove the reference to Profile? At minimum, Profile must remain in the solution for build to work successfully.